localization/english/wc_buildings_l_english.yml

```
@@ -22,11 +32,21 @@
  building_lordaeron_palace_01:1 "Lordaeron Palace"
  building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
  
+ building_type_nexus_01:0 "The Nexus"
+ building_type_nexus_01_desc:1 "The Nexus is an extensive series of caves and tunnels that riddles Coldarra."
+ building_nexus_01:1 "The Nexus"
+ building_nexus_desc:1 "The Nexus is an ancient ice fortress found in the center of Coldarra, an island in Borean Tundra. It is a column of magical energy surrounded by levitating earth-covered rings with ice caves underneath."
+ 
  building_type_ruined_palace_01:0 "Ruined Palace"
  building_type_ruined_palace_01_desc:1 "A city that spans beneath much of Lordaeron's ruin. A grave to kingdom."
  building_ruined_palace_01:1 "Ruined Palace"
  building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the once glorious city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a grim reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
  
+ building_type_shrine_of_fellowship_01:0 "Shrine Of Fellowship"
+ building_type_shrine_of_fellowship_01_desc:1 "A peaceful shrine up upon the peaks."
+ building_shrine_of_fellowship_01:1 "Shrine Of Fellowship"
+ building_shrine_of_fellowship_01_desc:1 "Many who are lost may wander down the road of the Path of Enlightenment. Up a set of leading stairs there lies a peaceful shrine. However, every so often a great duel of the dancers occurs here and many who come and partake realize, not all those who wander are lost."
```

```suggestion
 building_shrine_of_fellowship_01_desc:1 "Many who are lost may wander down the Path of Enlightenment and up a set of leading stairs where there lies a peaceful shrine. However, every so often a great duel of the dancers occurs here, and many who come to partake realize, that #italic not all those who wander are lost. #!"
```

Since path is a synonym for road, I think we can drop the "road" and just leave it at "Path of Enlightement".

Also made some slight changes to the sentence structure so that it flows a bit better (in my opinion, anyway) and I think it would be cool (but not necessary) to make the last phrase italic for added emphasis.